index.js

```diff
@@ -251,15 +240,23 @@ function BigAssFanAccessory(log, config) {
     return (value ? Characteristic.RotationDirection.COUNTER_CLOCKWISE : Characteristic.RotationDirection.CLOCKWISE);
   }
 
+  var occupancySetWrapper = function(value) {
```

Comment:
The reason I didn't have an occupancySetWrapper before was that it doesn't really make sense to set the occupancy reading on the device, since it's read only. Might be mistaken though